Runtime/Core/SplineHelpers.cs

```diff
@@ -17,7 +17,7 @@ public static class SplineHelpers
             float t, Vector3 p0, Vector3 p1, Vector3 p2, Vector3 p3)
         {
             t = Mathf.Clamp01(t);
-            return (-3f * p0 + 9f * p1 - 9f * p2 + 3f * p3) * t * t
+            return (-3f * p0 + 9f * p1 - 9f * p2 + 3f * p3) * (t * t)
```

Comment:
why is this faster?

Reply:
`((vector * float) * float)` vs `vector * (float * float)` -- this was actually pointed out by Rider and I auto-accepted it